content/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -5,53 +5,139 @@ weight = 50
 +++
 
 This page describes authentication for Kubeflow Pipelines to GCP.
+Available options listed below have different tradeoffs. You should choose the one that fits your use-case.
+
+* [Compute engine default service account](#compute-engine-default-service-account) is easy to set up, but overgrants permission. Therefore, it is not suitable for a shared GCP project.
+* [Workload Identity](#workload-identity) takes more efforts to set up, but allows fine-grained permission control. It is recommended for production use-cases.
+* [Google service account keys stored as Kubernetes secrets](#google-service-account-keys-stored-as-kubernetes-secrets) is the legacy approach and no longer recommended.
```

Comment:
no longer recommended in GKE. However in anthos/on-prem we still need it.

Reply:
thanks! will add that